src/App.jsx

```diff
@@ -1,104 +1,22 @@
 import './App.scss';
+import { PostList } from './components/PostList/PostList';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+const prepareData = () => {
```

Comment:
```suggestion
const preparePosts = () => {
```
Specify what kind of data you prepare